src/Ryujinx.Tests/Cpu/CpuTestSimdShImm32.cs

```diff
@@ -31,17 +31,17 @@ private static ulong[] _8B_()
         {
             return new[] { 0x0000000000000000ul, 0x7F7F7F7F7F7F7F7Ful, 0x8080808080808080ul, 0xFFFFFFFFFFFFFFFFul };
```

Comment:
```suggestion
            return new[] { 0x0000000000000000ul, 0x7F7F7F7F7F7F7F7Ful, 0x8080808080808080ul, 0xFFFFFFFFFFFFFFFFul, };
```
Same for above lines